test/app/lifecycle.coffee

```diff
@@ -154,7 +154,9 @@ describe 'lifecycle', ->
 
   describe '#ensureLoggedInOnAfterSignupPage', ->
 
-    it 'redirects to the login page without a user'
+    it 'redirects to the login page, and back, without a user', ->
```

Comment:
🤷‍♂️ not quite descriptive enough, but that's the intent. "When logged out, visiting personalize, it will redirect to login which will then redirect back to personalize"